index.php

```diff
@@ -98,7 +98,7 @@
       <div class="grid">
         <div class="two-thirds">
           <h2>What’s New in elementary OS 6 Odin</h2>
-          <p>The biggest update to the platform yet with multi-touch, dark style, app sandboxing, and an all-new installer. OS 6 empowers you to be in control and express yourself, continues to innovate with new features, and is both easier to get and more inclusive than ever.</p>
+          <p>The biggest update to date brings multi-touch, a perceptually dark style, app sandboxing, and an all-new installer. Odin grants visionary insight in the persuit of knowledge, with innovative features making the answers to riddles available to more people.</p>
```

Comment:
Norseman approved :)

Reply:
Eh, this changes language away from what's used in the blog post and changes the meaning.

Reply:
@cassidyjames I couldn't get through the blogpost while grasping content, some of which I know already.
If the concepts aren't there or stick, the info is not accessible or even retained.
The blogpost is an OK meeting, not something I would want to share with people.
It needs some editing and bite.